plugins/grpc/cpp/v1.64.1/buf.plugin.yaml

```diff
@@ -0,0 +1,11 @@
+version: v1
+name: buf.build/grpc/cpp
+plugin_version: v1.64.1
+source_url: https://github.com/grpc/grpc
+description: Generates C++ client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/cpp:v26.1
```

Comment:
This is still on 26.1 upstream: https://github.com/grpc/grpc/blob/d79949179d13140011ebc979b7ce5bfc79aaae8c/bazel/grpc_deps.bzl#L213-L227